sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -92,6 +92,19 @@ def test_decompress_compressed_no_header(http_request):
     except UnicodeDecodeError:
         pass
 
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
```

Comment:
@marcusc2 this should be an async call, and should be in `test_streaming_async.py`